systems/registry/member/github.com/ukama/ukama/systems/registry/network/pb/gen/network.validator.pb.go

```diff
@@ -0,0 +1,228 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
We need to fix the proto gen path so that generated files are place in the the right pb/ sub-folder  instead of created a new github.com/... tree.

Reply:
removed